web_widget_digital_signature/__openerp__.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2004-2010 OpenERP SA (<http://www.openerp.com>)
+#    Copyright (C) 2011-2015 Serpent Consulting Services Pvt. Ltd.
+#    (<http://www.serpentcs.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    "name": "Web Widget Digital Signature",
+    "version": "1.0",
+    "author": "Serpent Consulting Services Pvt. Ltd.",
+    "category": '',
+    'depends': ['web'],
+    "description": """
```

Comment:
Managed to do those changes!
On Tuesday 02 June 2015 03:46 PM, Markus Schneider wrote:
>
> In web_widget_digital_signature/__openerp__.py 
> <https://github.com/OCA/web/pull/105#discussion_r31508953>:
>
> > +#    but WITHOUT ANY WARRANTY; without even the implied warranty of
> > +#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
> > +#    GNU Affero General Public License for more details.
> > +#
> > +#    You should have received a copy of the GNU Affero General Public License
> > +#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
> > +#
> > +##############################################################################
> > +
> > +{
> > +    "name": "Web Widget Digital Signature",
> > +    "version": "1.0",
> > +    "author": "Serpent Consulting Services Pvt. Ltd.",
> > +    "category": '',
> > +    'depends': ['web'],
> > +    "description": """
>
> you can remove description if you have a readme.rst
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/OCA/web/pull/105/files#r31508953>.
>


-- 

Thanks,
Regards,

Serpent Consulting Services.
Web : http://www.serpentcs.com
Twitter: jaynvora, husendaudi
Skype: jaynvora, husen.daudi
Blog : http://www.serpentcs.com/blog